core/templates/dev/head/pages/topic_editor/main_editor/TopicEditorTabDirective.js

```diff
@@ -39,13 +39,19 @@ oppia.directive('topicMainEditor', [
             $scope.editableDescription = $scope.topic.getDescription();
             $scope.displayedTopicDescription = (
               $scope.editableDescription !== '');
+            $scope.firstTimeVisit = true;
           };
 
           var _initStorySummaries = function() {
             $scope.canonicalStorySummaries =
               TopicEditorStateService.getCanonicalStorySummaries();
           };
 
+          $scope.updateTopicDescriptionStatus = function(description) {
+            $scope.displayedTopicDescription = (description !== '');
+            $scope.firstTimeVisit = false;
```

Comment:
This seems weird to me. What is this firstTimeVisit thing? If it's the first time the user has visited the page, then I don't see why it gets updated when the "topic description status" is updated.

Maybe the variable name needs to be clearer.

Reply:
Will change the name. It was supposed to be to not show the error that description is empty, when the creator visited the topic editor for the first time.

Reply:
Done, renamed it.